ManhTien/05_ngay5_8/convert.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+
+
+class Convert
+{
+    public $numBer;
+    public function __construct($numBer)
+    {
+        return $this->numBer=$numBer;
+    }
+    
+}
+class Usd extends Convert
+{
+    public function UsdVnd()
+    {
+        return $this->numBer*23000;
```

Comment:
Đưa số `23000` thành 1 property đi e
```suggestion
        return $this->numBer * $this->usdToVnd;
```